LevelEditor/SliderWidget.py

```diff
@@ -0,0 +1,182 @@
+from PySide.QtCore import *
+from PySide.QtGui import *
+from collections import OrderedDict
+
+sliders = None
+
+
+class AdjustValues(QWidget):
+    valuesChanged = Signal(list)
+
+    def __init__(self, val_list, min_val, max_val):
+        super(AdjustValues, self).__init__()
+        self.val_list = val_list
+        self.min_val = min_val
+        self.max_val = max_val
+        self.slider_count = len(val_list)
```

Comment:
You shouldn't need to store this in an attribute; you can always just do `len(self.slides)` (which is O(1), so doesn't add any overhead).